test/tools/Modules/WebListener/WebListener.psm1

```diff
@@ -158,7 +158,8 @@ function Get-WebListenerUrl {
             return $null
         }
         $Uri = [System.UriBuilder]::new()
-        $Uri.Host = 'localhost'
+        # Use 127.0.0.1 and not localhost due to https://github.com/dotnet/corefx/issues/24104
```

Comment:
I'd add "TODO: remove after fix https://github.com/dotnet/corefx/issues/24104"

Reply:
why would we ever change it back? we are binding WebListener to 127.0.0.1, not to 'localhost'. it's more correct to keep it at 127.0.0.1.

Reply:
Good question! 😄 It seems IPv4 never die.

Reply:
At least we will have to make significant changes to the tests if we get rid of IPv4.  I was thinking that perhaps we should define a function to get the `value` for localhost.  Then, we only have one place to change, but this isn't a blocking comment.

Reply:
It could be done. for the most part.. the web cmdlets already do that since they get the url's from `Get-WebListenerUrl` the web listener module would just need to call whatever function we share with the rest of the tests. 

The rest of the tests in the project may not even benefit from this change. I did a quick look this morning and they were mostly `\\localhost\` (I'm not sure SMB is affected, the CoreFX issue seems to indicate it's specific to HTTP, some of those tests may need localhost regardless) or `WSMan:\localhost\` (which wouldn't change).

Reply:
In fact it's not worth.